include/sys/dataset_kstats.h

```diff
@@ -43,6 +46,8 @@ typedef struct dataset_kstat_values {
 	kstat_named_t dkv_nwritten;
 	kstat_named_t dkv_reads;
 	kstat_named_t dkv_nread;
+	kstat_named_t dkv_nunlinks;   /* num znodes added to unlinked set */
+	kstat_named_t dkv_nunlinked; /* num znodes processed by unlinked set */
```

Comment:
After giving this some more thought, I think it would be a little more useful and consistent if these were both monotonically increasing counters.  On mount `dkv_nunlinks` would be initialized to the unlinked set size and incremented whenever a new entry was added to the unlinked set.  `dkv_nunlinked` would be initialized to zero on mount and incremented when an entry was removed from the unlinked set.  The difference between the counters would still allow you to calculate the unlinked set size.  Plus you'd have the raw data to stuff in a database if you liked, or calculate things like the async unlink rate.